build.xml

```diff
@@ -87,6 +87,12 @@
         <copy file="${basedir}/LICENSE-2.0.txt" tofile="${zip-tmp}/${app.name}/LICENSE-2.0.txt"/>
         <copy file="${basedir}/NEWS.txt" tofile="${zip-tmp}/${app.name}/NEWS.txt"/>
         <copy file="${basedir}/KNOWN_ISSUES.txt" tofile="${zip-tmp}/${app.name}/KNOWN_ISSUES.txt"/>
+        <if>
+            <equals arg1="${os.family}" arg2="unix"/>
+            <then>
+                <copy file="${basedir}/unix_setup.sh" tofile="${zip-tmp}/${app.name}/unix_setup.sh"/>
+            </then>
```

Comment:
Do the replaceregexp here, on the copy. Then this file wont' be constantly changing and needing to be checked in each time we change the version.

Reply:
Move this one more line down.  Do the replace on the zip-tmp destination